lib/asyn-thread.c

```diff
@@ -144,9 +153,24 @@ static bool init_resolve_thread(struct Curl_easy *data,
                                 const char *hostname, int port,
                                 const struct addrinfo *hints);
 
+#if (_WIN32_WINNT >= 0x0600)
+/* Thread sync data used for GetAddrInfoExW  for win8+ */
+struct thread_sync_data_w8
+{
+    OVERLAPPED overlapped;
+    ADDRINFOEXW *res;
+    HANDLE cancel_ev;
+    WCHAR name[256]; /* max domain name is 253 chars */
+    WCHAR service_name[20]; /* port */
```

Comment:
This field is for holding the decimal 16 bit port number, 5 digits maximum (plus null byte). Why space for 20 wide chars?